plugin/trino-google-sheets/src/test/java/io/trino/plugin/google/sheets/TestGoogleSheets.java

```diff
@@ -78,4 +80,39 @@ public void testTableWithRepeatedAndMissingColumnNames()
         assertQuery("desc table_with_duplicate_and_missing_column_names", "SELECT * FROM (VALUES('a','varchar','','')," +
                 " ('column_1','varchar','',''), ('column_2','varchar','',''), ('c','varchar','',''))");
     }
+
+    // test polymorphic table function
+
+    @Test
+    public void testSheetQuerySimple()
+    {
+        assertQuery(
+                "SELECT * FROM TABLE(gsheets.system.sheet(id => '%s'))".formatted(DATA_SHEET_ID),
+                "VALUES " +
+                        "('1', 'one')," +
+                        "('2', 'two')," +
+                        "('3', 'three')," +
+                        "('4', 'four')," +
+                        "('5', 'five')");
+    }
+
+    @Test
+    public void testSheetQueryWithSheetTab()
+    {
+        assertQuery(
+                "SELECT * FROM TABLE(gsheets.system.sheet(id => '%s'))".formatted(DATA_SHEET_ID + "#number_text"),
```

Comment:
is `"#number_text"` the range? Could you add a test where the range affects the results?

Reply:
Sure, unfortunately this requires permissions to modify the existing testing google sheet and I do not have permissions to do so.

https://docs.google.com/spreadsheets/d/1S625j2oTptRepg6Yci68fCYE1269tdoSjljNOmTgQ3U/edit#gid=0

Reply:
I added more tests using the same sheet but with a different range filter. This tests the functionality without requiring modifying the sheet.